src/AssetConfig.php

```diff
@@ -180,7 +180,7 @@ public function load($path, $prefix = '')
             } elseif (strpos($section, self::FILTER_PREFIX) === 0) {
                 // filter section.
                 $name = str_replace(self::FILTER_PREFIX, '', $section);
-                $this->filterConfig($name, $values);
+                $this->filterConfig($name, array_merge($this->filterConfig($name), $values));
```

Comment:
Why are all those values different? The `.local` file is generally used when people are building and deploying to different platforms/os's. In which case wouldn't all those settings need to be different?